### PR TITLE
Contributing guide add

### PR DESCRIPTION
**Pull request by shivam-sharma7**
- Fixed issue #2

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+# Prerequisite Skills to Contribute
+- [Git](https://git-scm.com/) 
+- [node.js](https://nodejs.org/en/)
+
```
Comment (Shivansh-yadav13) [inline CONTRIBUTING.md:4]: @shivam-sharma7 also add [Docusaurus](https://docusaurus.io/)

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,5 @@
+# Prerequisite Skills to Contribute
+- [Git](https://git-scm.com/) 
+- [node.js](https://nodejs.org/en/)
```
Comment (AvineshTripathi) [inline CONTRIBUTING.md:3]: I believe we are not using nodejs for website currently, instead add React to it, javascript maybe 
cc @verma-kunal @pavangudiwada

**Comment by AvineshTripathi** [comment]
LGTM 

p.s. if anything needs to be added or removed, suggestions are always welcome

**Comment by verma-kunal** [comment]
Hey @shivam-sharma7 , this looks good! I do think we can make this a bit more informative for beginner folks out there, by adding more information like `how to commit`, `how to make a change`, `how to push` etc.
You can take reference from the guide [here](https://github.com/bradmccoydev/mentoring/blob/main/CONTRIBUTING.md) (from another repo) & see if you can add some more steps to our guide!

Let me know your thoughts on this or any questions :)
cc @AvineshTripathi @anubha-v-ardhan @saiyam1814

**Comment by AvineshTripathi** [comment]
do you want to add in this PR or have a seperate one

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -0,0 +1,99 @@
...
```
Comment (anubha-v-ardhan) [inline CONTRIBUTING.md:3]: I think we can safely remove this section.
Git is understood, we need it for any contribution and node js is not always required, the website repo will also contain DevOps roadmap notes which will be just markdown and major part of this project.
So, this section can be a bit misleading. Thoughts?

**Review by anubha-v-ardhan — CHANGES_REQUESTED** [review]

File: CONTRIBUTING.md (hunk 3 above)
Comment (shivam-sharma7) [reply CONTRIBUTING.md:3]: i am removing only node.js

Comment (AvineshTripathi) [reply CONTRIBUTING.md]: +1 to that @anubha-v-ardhan we can remove the section

**Comment by verma-kunal** [comment]
Hey @anubha-v-ardhan , is this PR good to go or have any other changes needs to be done?
cc @shivam-sharma7 @AvineshTripathi

**Review by anubha-v-ardhan — CHANGES_REQUESTED** [review]
Can we remove changes on README from this PR as it's already being handled by #36. Committing changes on README via this PR will create conflicts.

**Comment by anubha-v-ardhan** [comment]
> Hey @anubha-v-ardhan , is this PR good to go or have any other changes needs to be done? cc @shivam-sharma7 @AvineshTripathi

I was thinking if we can do a more detailed contributing guide diviiding it into two sections for core website work and notes work since notes will also be on website repo. But since it's not clear yet how the notes work will proceed, I think we can merge this once https://github.com/kubesimplify/website/pull/26#pullrequestreview-972992673 is resolved and make issue for enhancing this guide later.

**Comment by anubha-v-ardhan** [comment]
@shivam-sharma7 Please comment on #2 and get it assigned to yourself to avoid confusion. Always notify everyone by commenting on the issue before you start working on it.

**Pull request closed without merging**